packages/web-components/docs/api-report.md

```diff
@@ -222,6 +228,8 @@ export class FluentDesignSystemProvider extends DesignSystemProvider implements
     accentPalette: string[];
     backgroundColor: string;
     // (undocumented)
+    protected backgroundColorChanged(): void;

```

Comment:
Not sure if it will apply here but we have some eslint rule to avoid using visibility modifiers. https://github.com/microsoft/fluentui/blob/master/packages/eslint-plugin/README.md#no-visibility-modifiers

Reply:
I don't know why this file was changed, I think it was done automatically by yarn. Is it possible?